### DISCUSSION TITLE
Additional control variables

### DISCUSSION THREAD

**Discussion by lightsociologist** [lightsociologist/hirsch_test#4] [General]
@AaronGullickson One idea that was embedded in my edits of your analysis script was the inclusion of two additional control variables - a control for institutional/university volume and a control for interdisciplinarity). The bibliometricians sought some additional controls when the paper was reviewed. There are obviously limits here and sociologists may have less of a concern. What do you think?

**Comment by AaronGullickson** [comment]
So those variables are already in the dataset? On some level, I am not sure how much controls matter here for the story. Seems like a case where you could easily overcontrol and end up controlling for mediators rather than confounders, but its probably worthwhile to at least test whether it makes much difference.

**Comment by AaronGullickson** [comment]
Since this seems pretty actionable, lets make it an issue.
